gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=dev.misfitlabs.kotlinguice4
-VERSION_NAME=1.7.0-SNAPSHOT
+VERSION_NAME=2.0.0-SNAPSHOT
```

Comment:
Since Guice did a 6.0 and a 7.0 release simultaneously, should this go to 3.x for the 7.0 Guice or is 6.0 backwards compatible enough that it could be used with 1.x?

Reply:
I haven't looked into 6 much at all; this could go to 3.x and a separate branch could go to 2.x to support 6, but i've only looked into jumping straight from 5 to 7 and trying to get the whole jakarta transition over with all at once.  i'd guess 6 is compatible enough to work with 1.x while staying with javax.* but haven't verified.

Reply:
ok, took a little time to look into it, it appears that using guice 6.0.0 with jakarta.inject 2.0.1 and javax.inject 1 all on the classpath works fine with kotlin-guice 1.6.0 for existing supported javax namespaced inject annotations.  

arguably a kotlin-guice 2.x that works with guice 6 and adds support for jakarta namespaced inject annotations (ie supports both javax & jakarta namespaces, with the caveats from the guice 6 release notes on providers etc) might scratch somebody's itch, but doing that and then removing the javax namespaced inject annotations in kotlin-guice 3.x seems unnecessary to me.  up to you how you want to proceed.

Reply:
Just to be safest and to best match the way Guice did the upgrade, I'm going to bump up to 3.x for the jarkata change and keep 2.x open if there is demand for a transition release using Guice 6.x.